src/components/Motivation.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../index.css';
+
+import img1 from '../images/Motivation-1.png'
+import img2 from '../images/Motivation-2.png'
+import img3 from '../images/Motivation-3.png'
+import icon from '../images/Read-more.png'
+
+// eslint-disable-next-line max-len
+const Motivation = ({ title, backgroundImg, img1Title, img2Title, img3Title, img1Subtitle, img2Subtitle, img3Subtitle, shortText }) => {
+  const motivationBackground = {
+    backgroundImage: `url(${backgroundImg})`
+  };
+
+  return (
+    <section className="motivation-section" style={motivationBackground}>
+      <div className="motivation-container">
+        <h2>{title}</h2>
+        <div className="images-container">
+          <div className="img-container-motiv">
+            <img src={img1} alt={img1Title} />
```

Comment:
You might want to consider adding alt text that describes what the images look like.